site/content/blog/2020-10-01-whats-new-in-svelte-october-2020.md

```diff
@@ -0,0 +1,59 @@
+---
+title: What's new in Svelte: October 1, 2020
+description: New object methods, in-depth learning resources and tons of integration examples!
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+## New features
```

Comment:
Something about `$$slots` should be in here! It's a pretty dope feature! 👍

Reply:
Something about the new `createEventDispatcher` typing for Typescript users and how to use it and its advantages would be cool. I can add a suggestion paragraph tomorrow if you want.

Reply:
There were a few fixes for await blocks too, they could be gathered together with a short description perhaps?